src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTestForDisabledService.java

```diff
@@ -75,4 +81,36 @@ public void should_reject_file_when_service_is_disabled() throws Exception {
         errorWasSent(SAMPLE_ZIP_FILE_NAME, ErrorCode.ERR_SERVICE_DISABLED);
     }
 
+    @Test
+    public void should_not_move_file_if_send_message_throws_exception_when_service_is_disabled() throws Exception {
```

Comment:
How about something more general, e.g. `should_abandon_rejecting_file_when_sending_error_notification_fails`? The reasons of why we're trying to reject the file is irrelevant here.

Reply:
Done. And new test class created